backend/src/main/kotlin/com/example/demo/ServletInitializer.kt

```diff
@@ -0,0 +1,12 @@
+//package com.example.demo
+//
+//import org.springframework.boot.builder.SpringApplicationBuilder
+//import org.springframework.boot.web.servlet.support.SpringBootServletInitializer
+//
+//class ServletInitializer : SpringBootServletInitializer() {
+//
+//	override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
+//		return application.sources(DemoApplication::class)
+//	}
+//
+//}
```

Comment:
@XIC557-RamaKrishna can you explain why you decided not to delete these files, but fully comment them instead?